sdcm/argus_results.py

```diff
@@ -93,10 +92,15 @@ class Meta:
         name = "Restore benchmark"
         description = "Restore benchmark"
         Columns = [
-            ColumnMetadata(name="restore time", unit="s", type=ResultType.DURATION),
-            ColumnMetadata(name="repair time", unit="s", type=ResultType.DURATION),
-            ColumnMetadata(name="total", unit="s", type=ResultType.DURATION),
+            ColumnMetadata(name="restore time", unit="s", type=ResultType.DURATION, higher_is_better=False),
+            ColumnMetadata(name="repair time", unit="s", type=ResultType.DURATION, higher_is_better=False),
+            ColumnMetadata(name="total", unit="s", type=ResultType.DURATION, higher_is_better=False),
         ]
+        ValidationRules = {
```

Comment:
@mikliapko please review if these numbers seem reasonable - 10% longer duration than 'best result' will mark test as failed.

Reply:
Looks reasonable to me